exemplar-crd/go.mod

```
@@ -0,0 +1,10 @@
+module github.com/k8s-service-bindings/examplar-crd
```

nit: align module with git URL. Technically the module could be anything since it should never be consumed externally.

```suggestion
module github.com/k8s-service-bindings/spec/examplar-crd
```

---

Resolved by a62ba29